README.md

```diff
@@ -1,4 +1,9 @@
+Install + Run with Moreh 
```

Comment:
It also should include conda create/activate/deactivate before installing dependencies.

Reply:
I added the conda create and activate steps in the latest commit